frontend/src/actions/groups/tags.js

```diff
@@ -0,0 +1,32 @@
+import { get } from '../../lib/api';
+import * as constants from '../../constants/groups';
+
+export default () => {
+  return dispatch => {
+    dispatch(request());
+
+    return get('/groups/tags')
+    .then(json => dispatch(success(json)))
+    .catch(error => dispatch(failure(error)));
+  };
+};
+
+function request() {
+  return {
+    type: constants.GROUP_TAGS_REQUEST
+  };
+}
+
+export function success(json) {
+  return {
+    type: constants.GROUP_TAGS_SUCCESS,
+    json
+  };
+}
+
+function failure(error) {
+  return {
+    type: constants.GROUP_TAGS_FAILURE,
+    error
+  };
+}
```

Comment:
We only need to know the Group tags available to filter by **once** on every page load. 

The Discover action will be called at least once on every page load, however as the user scrolls down new results will be fetched, the Discover action will be called **multiple** times as the user discovers more results with the same filter values. Changing filter values will also trigger the Discover action again.